internal/cmd/egctl/utils.go

```diff
@@ -79,3 +85,36 @@ func newK8sClient() (client.Client, error) {
 
 	return cli, nil
 }
+
+// ClosePortForwarderOnInterrupt closes the port forwarder when an interrupt signal is received
+func ClosePortForwarderOnInterrupt(fw kube.PortForwarder) {
+	go func() {
+		signals := make(chan os.Signal, 1)
+		signal.Notify(signals, os.Interrupt)
+		defer signal.Stop(signals)
+		<-signals
+		fw.Stop()
+	}()
+}
+
+// openBrowser opens the given URL in the default browser
+func openBrowser(url string, writer io.Writer) {
```

Comment:
can we reuse from https://github.com/envoyproxy/gateway/blob/main/internal/cmd/egctl/dashboard_envoy.go

Reply:
Actually, it is reusing that, but move it into utils, since two places need this func